INSTALL

```diff
@@ -18,7 +18,9 @@ higher.
    * libxml2 (RECOMMENDED)
    * libcurl (RECOMMENDED)
    * libdispatch (RECOMMENDED)
-   * libxslt (OPTIONAL)
+   * libavahi (RECOMMENDED for NSNetServices)
+   * libxslt (RECOMMENDED)
```

Comment:
Why was this changed from `OPTIONAL` to `RECOMMENDED`?

Reply:
GNUstep extensions are using zlib for compression support, and OPTIONAL
was used in a context where there are different options (i.e. gnutls and openssl, or iconv and glibc).

Having an incomplete installation is just wrong imo.

Reply:
NSXMLDocument is using libxslt for methods like objectByApplyingXSLT:arguments:error: and return nil if gnustep-base
was compiled without libxslt (See https://github.com/gnustep/libs-base/blob/5f92c87d784753c996e14e4c75af7464014b6833/Source/NSXMLDocument.m#L429C8-L429C28).

Having an incomplete installation is just wrong imo.

OPTIONAL is used in a context where there are different options (i.e. gnutls and openssl, or iconv and glibc).

Reply:
I see, this is public API and available since macOS 10.4.

Reply:
I think we generally need to improve toolchain consistency by enforcing certain libraries during the configuration phase.